vcpkg/readme/vcpkg-README.md

```diff
@@ -0,0 +1,347 @@
+---
+title: Microsoft/vcpkg README
+description: README for vcpkg registry
+author: JavierMatosD
+ms.author: javiermat
+ms.date: 10/17/2023
+ms.prod: vcpkg
+ROBOTS: NOINDEX
+---
+
+# Vcpkg: Overview
+
+vcpkg helps you manage C and C++ libraries on Windows, Linux and MacOS.
+This tool and ecosystem are constantly evolving, and we always appreciate contributions!
+
+If you've never used vcpkg before, or if you're trying to figure out how to use vcpkg,
+check out our [Getting Started](#getting-started) section for how to start using vcpkg.
```

Comment:
```suggestion
If you've never used vcpkg before, or if you're trying to figure out how to use vcpkg,
check out our [guide to start using vcpkg](../get_started/get-started.md).
```

We should link to the `Get Started` guide on Learn instead of having the same content on this file.